New version: Albion.Online version 24.8.21

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #169421

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/169431)

Dragon1573 committed:
New version: Albion.Online version 20240821101118

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

Dragon1573 commented:
@AsciiWolf, it seems `albiononline.com` now forbid all UA requests except normal browsers.

```text
23:43:56 C:\...\CMake 1ms pwsh> Invoke-RestMethod -Method Get -Uri "https://albiononline.com/home"      
Invoke-RestMethod: Just a moment...*{box-sizing:border-box;margin:0;padding:0}html{line-height:1.15;
# Following contents are ignored
```

Dragon1573 committed:
Update version: Albion.Online version 20240821101118

- Add `MinimumOSVersion` of the package

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

Dragon1573 committed:
Move version (Albion.Online): 20240821101118 ==> 24.8.21

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

stephengillie commented on manifests/a/Albion/Online/24.8.21/Albion.Online.locale.en-US.yaml:
```diff
@@ -0,0 +1,21 @@
+PackageVersion: 24.8.21
+PackageLocale: en-US
+Publisher: Sandbox Interactive GmbH
+PublisherUrl: https://albiononline.com/
+PrivacyUrl: https://albiononline.com/privacy_policy
+PackageName: Albion Online
+PackageUrl: https://albiononline.com/download
+License: Proprietary
+LicenseUrl: https://albiononline.com/en/terms_and_conditions
```
> Url validation error
> 
> * manifests\a\Albion\Online\24.8.21
>   * https://albiononline.com/
>     * Http status code: Forbidden
>   * https://albiononline.com/privacy_policy
>     * Http status code: Forbidden
>   * https://albiononline.com/download
>     * Http status code: Forbidden
>   * https://albiononline.com/en/terms_and_conditions
>     * Http status code: Forbidden
```suggestion
# PublisherUrl: 
# PrivacyUrl: 
PackageName: Albion Online
# PackageUrl: 
License: Proprietary
# LicenseUrl:
```

AsciiWolf replied:
Or just waive these errors. The urls are all correct and will work fine when an user clicks them.

Dragon1573 replied:
I'd prefer keeping them in the manifest, but add comment them out to prevent validation errors.

```suggestion
# PublisherUrl: https://albiononline.com/
# PrivacyUrl: https://albiononline.com/privacy_policy
PackageName: Albion Online
# PackageUrl: https://albiononline.com/download
License: Proprietary
# LicenseUrl: https://albiononline.com/en/terms_and_conditions
```

Dragon1573 committed:
Apply suggestions from code review

- "Albion Online" server prevents connections except human browsers.
- Just comment them out to prevent validation errors.
- Keep them in manifests for further contributors & volunteers.
